backend/app/controllers/spree/admin/orders_controller.rb

```diff
@@ -152,16 +152,16 @@ def resend
       end
 
       def open_adjustments
-        adjustments = @order.all_adjustments.where(state: 'closed')
-        adjustments.each &:open!
+        adjustments = @order.all_adjustments.closed
```

Comment:
Should we also deprecate the old scope and add `finalized` and `not_finalized` scopes instead? I find it slightly confusing that this is the one place where we still use `closed` and `open`.